opencti-platform/opencti-front/src/private/components/data/ToolBar.js

```diff
@@ -236,6 +236,8 @@ const toolBarConnectorsQuery = graphql`
   }
 `;
 
+const maxNumberOfObservablesToCopy = 4;
+
 class ToolBar extends Component {
```

Comment:
I thought we said that you had already done a good work on getting this file into pure function JS and that we should use it ?